.github/workflows/build-docker-manifest.yaml

```
@@ -0,0 +1,87 @@
+        uses: hoprnet/hopr-workflows/actions/multi-arch-manifest@ausias/fix-actions
+        with:
+          registry: ${{ inputs.docker_gcp_registry }}
+          image: ${{ inputs.docker_image_name }}
+          version: latest
+      - name: Log in to Docker Hub
+        if: inputs.version_type == 'release'
+        uses: docker/login-action@5e57cd118135c172c3672efd75eb46360885c0ef # v3.6.0
+        with:
+          username: ${{ secrets.docker_hub_username }}
+          password: ${{ secrets.docker_hub_token }}
+      - name: Create multi-arch docker manifest - Docker Hub - ${{ inputs.docker_version }}
```

actionlint (1.7.9)

property "docker_version" is not defined in object type {docker_build_version: string; docker_debug_version: string; docker_gcp_registry: string; docker_hub_registry: string; docker_image_name: string; version_type: string}

(expression)